CONTRIBUTING.md

```diff
@@ -329,18 +329,11 @@ Behind the scenes, `templatekit` uses [Jinja](http://jinja.pocoo.org) and [Cooki
 To install `templatekit` for development, run
 
 ```bash
-pip install -e ".[dev]"
+pip3 install -e templatekit 
```

Comment:
templatekit is not longer developed in the lsst/templates repository. That's why this section got outdated.

Instead of this misleading change, delete the `## template` section entirely. Or I can clean it up after this PR lands.

Reply:
removed